### PR TITLE
Add Cartesian goal support to station keeping

### PR DESCRIPTION
**Pull request by mabelzhang**
Supports specifying the goal in Cartesian coordinates in `goal_pose_cart` parameter.

### CODE REVIEW

File: vrx_gazebo/src/stationkeeping_scoring_plugin.cc [hunk 1]
```diff
@@ -43,36 +43,69 @@ void StationkeepingScoringPlugin::Load(gazebo::physics::WorldPtr _world,
   gzmsg << "Task [" << this->TaskName() << "]" << std::endl;
 
   // Get lat, lon and yaw from SDF
-  ignition::math::Vector3d latlonyaw(0, 0, 0);
-  if (!_sdf->HasElement("goal_pose"))
+  if (!_sdf->HasElement("goal_pose") && !_sdf->HasElement("goal_pose_cart"))
   {
     ROS_ERROR("Unable to find <goal_pose> element in SDF.");
```
Comment (caguero) [inline vrx_gazebo/src/stationkeeping_scoring_plugin.cc:48]: `<goal_pose>` or `<goal_pose_cart>`?

**Review by caguero — APPROVED** [review]

File: vrx_gazebo/src/stationkeeping_scoring_plugin.cc (hunk 1 above)
Comment (mabelzhang) [reply vrx_gazebo/src/stationkeeping_scoring_plugin.cc]: Rephrased 31932b3

**Comment by mabelzhang** [comment]
31932b3 added lines to install the headers of `vrx_gazebo`, and export `scoring_plugin` library for the consumption in VORC.

**Review by caguero — APPROVED** [review]

**Comment by caguero** [comment]
> [31932b3](https://github.com/osrf/vrx/commit/31932b337c2a013d813655c76b815df30085ad4a) added lines to install the headers of `vrx_gazebo`, and export `scoring_plugin` library for the consumption in VORC.

Looks good!

**Pull request merged**